src/together/resources/finetune.py

```diff
@@ -35,7 +37,10 @@ def createFinetuneRequest(
     n_checkpoints: int | None = 1,
     batch_size: int | Literal["max"] = "max",
     learning_rate: float | None = 0.00001,
+    min_lr_ratio: float | None = 0.0,
```

Comment:
Can we define it simply as float, since this should be always defined by that point?

Reply:
Done. Also changed for max_grad_norm and weight_decay